Header header
string robot_name
string control_type
string[] joint_names
float64[] positions
float64[] velocities
float64[] torques
